wiki/advantages-and-features.md

```diff
@@ -19,8 +19,6 @@ While there are some other cryptocurrencies that try and reward scientific work,
 they are limited in other various ways (most are heavily centralized among other 
 things) see:
 * [Comparison to CureCoin](#comparison-to-curecoin "wikilink")
-* [Comparison to Obyte](#comparison-to-obyte "wikilink")
-* [Comparison to Primecoin](#comparison-to-primecoin "wikilink")
```

Comment:
What is your source that Primecoin is dead? Their website still works and the chain is still producing blocks.

Reply:
You're right, my bad I thought they were dead, added back